[2020-10-29 14:48:08,307] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: Currency.create_db 2020-10-27T00:00:00+00:00 [queued]>
[2020-10-29 14:48:08,317] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: Currency.create_db 2020-10-27T00:00:00+00:00 [queued]>
[2020-10-29 14:48:08,317] {taskinstance.py:880} INFO - 
--------------------------------------------------------------------------------
[2020-10-29 14:48:08,317] {taskinstance.py:881} INFO - Starting attempt 1 of 2
[2020-10-29 14:48:08,317] {taskinstance.py:882} INFO - 
--------------------------------------------------------------------------------
[2020-10-29 14:48:08,323] {taskinstance.py:901} INFO - Executing <Task(PythonOperator): create_db> on 2020-10-27T00:00:00+00:00
[2020-10-29 14:48:08,326] {standard_task_runner.py:54} INFO - Started process 6289 to run task
[2020-10-29 14:48:08,350] {standard_task_runner.py:77} INFO - Running: ['airflow', 'run', 'Currency', 'create_db', '2020-10-27T00:00:00+00:00', '--job_id', '292', '--pool', 'default_pool', '--raw', '-sd', 'DAGS_FOLDER/currency.py', '--cfg_path', '/var/folders/4j/yctg8njd19g2873bfk914jqw0000gn/T/tmp6rfohsn0']
[2020-10-29 14:48:08,352] {standard_task_runner.py:78} INFO - Job 292: Subtask create_db
[2020-10-29 14:48:13,380] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: Currency.create_db 2020-10-27T00:00:00+00:00 [running]> Arpitas-MacBook-Pro.local
[2020-10-29 14:48:18,399] {base_hook.py:89} INFO - Using connection to: id: sqlite2. Host: /Users/arpitasaha/airflow/dags/db/DestinationDB.db, Port: None, Schema: /Users/arpitasaha/airflow/dags/db/DestinationDB.db, Login: None, Password: None, extra: None
[2020-10-29 14:48:18,401] {python_operator.py:114} INFO - Done. Returned value was: None
[2020-10-29 14:48:18,405] {taskinstance.py:1070} INFO - Marking task as SUCCESS.dag_id=Currency, task_id=create_db, execution_date=20201027T000000, start_date=20201029T091808, end_date=20201029T091818
[2020-10-29 14:48:23,378] {local_task_job.py:102} INFO - Task exited with return code 0
